source/system-administrators/activities/autoclustering-with-studio.rst

```diff
@@ -216,26 +216,35 @@ You can also verify from Studio that there are two nodes in the cluster by click
 Setup the Studio Arbiter
 ------------------------
 
-Whenever the number of Studios in the cluster is even numbered, the Studio Arbiter needs to be started.  To setup the Studio Arbiter:
+Whenever the number of Studios in the cluster is an even numbered, the Studio Arbiter needs to be started.  
+To setup the Studio Arbiter:
 
-#. Configure the Arbiter by configuring the following items in your ``bin/studio-arbiter/config/application.yaml`` file
+.. note:: Studio Arbiter can't run on any authoring server because it needs to use the same ports that Studio.
```

Comment:
Typo? Maybe add `uses` at the end so the sentence goes like: `it needs to use the same ports that Studio uses.` or something like that.